src/objective-c/GRPCClient/GRPCCall.m

```diff
@@ -92,7 +93,7 @@ @interface GRPCCall () <GRXWriteable>
 @implementation GRPCCall {
   dispatch_queue_t _callQueue;
```

Comment:
Then let's adjust the explanation on top of line 93 appropriately, and reevaluate the use of _callQueue.